java/Ice/greeter/client/src/main/java/com/zeroc/greeter/Client.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) ZeroC, Inc.
+
+package com.zeroc.greeter;
+
+import com.zeroc.Ice.Communicator;
+import com.zeroc.Ice.Util;
+import com.zeroc.greeter.VisitorCenter.GreeterPrx;
+import java.util.concurrent.CompletableFuture;
+
+class Client {
```

Comment:
I feel like this should probably be public, since it's not an internal implementation detail or anything.
In Java, putting no access qualified on a class is equivalent to `internal` in C#.
```suggestion
public class Client {
```